<?xml version="1.0"?>
<launch>

	<arg name="current_mission_state" default="0" />
    	<arg name="is_dynamic_finished" default="false" />
    	<arg name="is_parking_dist" default="3.0" />  <!-- 밀릴때 3.2 안밀릴때 3.0 -->
	
	
	<node name="waypoint_follower_node" pkg="waypoint_follower" type="waypoint_follower" output="screen">
		<param name="current_mission_state" value="$(arg current_mission_state)" />
		<param name="is_parking_dist" value="$(arg is_parking_dist)" />
        <param name="is_dynamic_finished" value="$(arg is_dynamic_finished)" />

	</node>

</launch>
